package.json

```diff
@@ -320,19 +320,13 @@
         "title": "Run Source",
         "category": "R",
         "command": "r.runSource",
-        "icon": {
-          "light": "./images/DownloadFile.ico",
-          "dark": "./images/DownloadFile_inverse.ico"
-        }
+        "icon": "$(run)"
       },
       {
         "title": "Knit Rmd",
         "category": "R",
         "command": "r.knitRmd",
-        "icon": {
-          "light": "./images/DownloadFile.ico",
```

Comment:
These ico files seem to be no longer used anywhere. Should we remove them?

Reply:
OK, thank you for you review!